inventory/host_vars/openfoodnetwork.org.au/config.yml

```diff
@@ -28,8 +28,8 @@ postgres_listen_addresses:
 custom_hba_entries:
   - "{{ custom_hba_metabase }}"
   - "{{ custom_hba_n8n }}"
-  - { type: hostssl, database: "{{ db }}", user: zapier, address: '54.86.9.50/32', auth_method: md5 }
-  - { type: hostssl, database: "{{ db }}", user: zapier, address: '167.99.89.242/32', auth_method: md5 } # for metabase (data.openfoodnetwork.org.au)
```

Comment:
I think that we still need these. Definitely Metabase. Zapier is also in use but I don't know if we use the database connection.

Reply:
I re enable Zappier for those that weren't commented out. Sorry I was under the impression we moved to N8N and were not using Zappier any more.